desktop-src/direct3dhlsl_current/hlsl-method-sample.md

```diff
@@ -0,0 +1,226 @@
+# Sample
+
+Samples a texture with optional parameters for: immediate offset, level of detail clamp, residency status output.  Cube textures do not support immediate offsets.
+
+Method Signatures vary per supported TextureObject type:
+
+### Signatures
+
+#### Texture1D
+```syntax
+ElementType Texture1D<ElementType>::Sample(
+      in  SamplerState  samp,
+      in  float         coord
+   [, in  int           offset
+   [, in  float         lodClamp
+   [, out uint          status ] ] ]
+);
+```
+
+#### Texture1DArray
+```syntax
+ElementType Texture1DArray<ElementType>::Sample(
+      in  SamplerState  samp,
+      in  float2        coord
+   [, in  int           offset
+   [, in  float         lodClamp
+   [, out uint          status ] ] ] );
+```
+
+#### Texture2D
+```syntax
+ElementType Texture2D<ElementType>::Sample(
+      in  SamplerState  samp,
+      in  float2        coord
+   [, in  int2          offset
+   [, in  float         lodClamp
+   [, out uint          status ] ] ] );
+```
+
+#### Texture2DArray
+```syntax
+ElementType Texture2DArray<ElementType>::Sample(
+      in  SamplerState  samp,
+      in  float3        coord
+   [, in  int2          offset
+   [, in  float         lodClamp
+   [, out uint          status ] ] ] );
+```
+
+#### Texture3D
+```syntax
+ElementType Texture3D<ElementType>::Sample(
+      in  SamplerState  samp,
+      in  float3        coord
+   [, in  int3          offset
+   [, in  float         lodClamp
+   [, out uint          status ] ] ] );
+```
+
+#### TextureCube
+```syntax
+ElementType TextureCube<ElementType>::Sample(
+      in  SamplerState  samp,
+      in  float3        coord
+   [, in  float         lodClamp
+   [, out uint          status ] ] ] );
```

Comment:
typo: extra square close bracket `]`.  Same with next method signature.

Reply:
Thank you.  Corrected.